py3status/modules/arch_updates.py

```diff
@@ -1,120 +1,94 @@
 # -*- coding: utf-8 -*-
-
 """
 Display number of pending updates for Arch Linux.
 
-This will display a count of how many 'pacman' updates are waiting
-to be installed and optionally a count of how many 'aur' updates are
-also waiting.
-
 Configuration parameters:
-    cache_timeout: How often we refresh this module in seconds (default 600)
-    format: Display format to use
-        (default 'UPD: {pacman}' or 'UPD: {pacman}/{aur}')
-    hide_if_zero: Don't show on bar if True
-        (default False)
-    include_aur: Set to True to use 'cower' to check for AUR updates
-        (default False)
+    cache_timeout: refresh interval for this module (default 600)
+    check_aur: check aur for pending updates (default None)
+    check_pacman: check pacman for pending updates (default None)
+    format: display format for this module (default '{updates}')
+    format_aur: (default 'UPD: {aur}')
+    format_full: (default 'UPD: {pacman}/{aur}')
+    format_pacman: (default 'UPD: {pacman}')
```

Comment:
Uses single `format` now, but not working as intended.

Is there something to make it false/skip if zero number?